Makefile

```diff
@@ -179,6 +179,12 @@ mdlint:
 	ruby:slim \
 	/workdir/scripts/test-infra/mdlint.sh
 
+.PHONY: helm-schema
+helm-schema:
+	# Requires helm plugin install https://github.com/losisin/helm-values-schema-json.git
+	cd deployment/helm/node-feature-discovery/ && \
+	helm schema
+
```

Comment:
Do you think it is a bad idea to actually install the plugin here directly so that we allow local validations as well?

Reply:
`check if the plugin is installed and print instructions`... sounds good

Reply:
> `check if the plugin is installed and print instructions`... sounds good

Thanks, done